### PR TITLE
docs: Update docs for publishing

### PR DESCRIPTION
**Pull request by tumido**
Before demoing this app at Kubecon, we need to sort out some things... Some content was just moved around, other sourced from related communities and adjusted to our needs.

Resolves: #65 #64 #67 #68

/cc @harshad16 @mimotej @KPostOffice 

please review

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+* Search to find out if a similar question has been asked
+* Try to define what you need help with:
+  * Is there something in particular you want to do?
+  * What problem are you encountering and what steps have you taken to try and fix it?
+  * Is there a concept you don’t understand?
+* Provide sample code or a video, if possible
+* Screenshots can help, but if there’s important text such as code or error messages in them, please also provide those as text
+* The more time you put into asking your question, the better we can help you
+
+## Contributions
```
Comment (harshad16) [inline SUPPORT.md:28]: how about we move this to the readme page? it would be easy for contributors in that way.

**Review by harshad16 — COMMENTED** [review]
/lgtm
This is amazing :100: 
just one comment.

File: SUPPORT.md (hunk 1 above)
Comment (tumido) [reply SUPPORT.md:28]: Good call. What if we leave it here and add a similar note to README, wdyt @harshad16 ? I think it makes sense to have it it both places. :slightly_smiling_face:

Comment (harshad16) [reply SUPPORT.md:28]: that would be great as well :)

**Review by harshad16 — COMMENTED** [review]
Thanks :100: 
/lgtm

will wait for one more reviewer to review and will merge this.

**Review by KPostOffice — COMMENTED** [review]
/lgtm ++

**Pull request merged**